Source/UI/About/AliasCountButton.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  AliasCountButton.swift
+//  Planetary
+//
+//  Created by Chad Sarles on 12/19/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import SwiftUI
+
+struct AliasCountButton: View {
+    
+    var aliases: [RoomAlias]
+    var count: Int
+    
+    @State
+    private var showingAliases = false
+    
+    private func aliasList(_ list: [RoomAlias], isPresented: Binding<Bool>) -> some View {
+        NavigationView {
+            AliasListView(aliases: aliases)
+                .navigationTitle("\(aliases.count) \(Localized.Alias.aliases.text)" )
+                .toolbar {
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        Button {
+                            isPresented.wrappedValue = false
+                        } label: {
+                            Image.navIconDismiss
+                        }
+                    }
+                }
+                .navigationBarTitleDisplayMode(.inline)
+        }
+    }
+
+    var body: some View {
+        Button {
+            showingAliases = true
+        } label: {
+            Text("+\(count)")
+                .foregroundLinearGradient(
+                    LinearGradient(
```

Comment:
Maybe you could add this gradient to `LinearGradient+Planetary.swift`, or maybe it's already there?

Reply:
Same in `AliasListView`.